src/writer.js

```diff
@@ -18,6 +18,17 @@ class Writer {
     return this._queue.length
   }
 
+  printableJsonTrace (formattedTrace) {
```

Comment:
I would prefer a verb here. Also, it shouldn't be a public method.

I also wonder if it would be possible to have the actual Uint64BE instance return the correct string instead?

Reply:
Looks like it's because of `valueOf`: https://github.com/kawanet/int64-buffer#output-methods